app/src/components/common/icons/IconClose.tsx

```diff
@@ -34,11 +34,11 @@ export const IconClose = (props: Props) => {
       viewBox="0 0 24 24"
       width={size}
       xmlns="http://www.w3.org/2000/svg"
+      {...restProps}
     >
       <path
         className="path"
         d="M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z"
-        fill={hoverEffect ? '#DCDFF2' : color}
```

Comment:
Because you have deleted this and just left `fill: ${props => props.theme.colors.textColorDark};` in the wrapper above we get this issue...
![Screenshot 2021-07-12 at 18 16 32](https://user-images.githubusercontent.com/33226956/125321776-53995500-e33d-11eb-9cc3-8aa50cca8660.png)
Where we get this Dark IconClose.. @Frankaus Did geronimo suggest that change?

Reply:
Please compare master and your pr to make sure all is the same...

Reply:
Mhhh no Geronimo didn't request any change. My bad yes, it was tertiary! I'll fix that tmrw cause I was today on the review so I don't create more issues with branching etc
![Screenshot 2021-07-12 at 18 31 53](https://user-images.githubusercontent.com/57708955/125323753-6f9df600-e33f-11eb-97b3-975b90af8cd3.png)